src/pollution_model/run_medusa_2.py

```diff
@@ -1,474 +1,520 @@
+        curr_dissolved_copper, curr_dissolved_zinc = dissolved_metal_load(total_copper_load=curr_total_copper,
+                                                                          total_zinc_load=curr_total_zinc,
+                                                                          surface_type=surface_type)
+        all_buildings.loc[index] = [index, surface_area, surface_type, curr_tss,
+                                    curr_total_copper, curr_total_zinc, curr_dissolved_copper, curr_dissolved_zinc]
+
+    # Run through all the roads/car parks, and calculate TSS, total metal loads, and dissolved metal loads
+    for i in range(len(all_roads)):
+        surface_area = float(all_roads.iloc[i]["SurfaceArea"])
+        surface_type = all_roads.iloc[i]["SurfaceType"]
+        curr_tss = compute_tss_roof_road(surface_area=surface_area, antecedent_dry_days=antecedent_dry_days,
+                                         average_rain_intensity=average_rain_intensity, event_duration=event_duration,
+                                         surface_type=surface_type)
+
+        curr_total_copper, curr_total_zinc = total_metal_load_road_carpark(curr_tss)
+        curr_dissolved_copper, curr_dissolved_zinc = dissolved_metal_load(total_copper_load=curr_total_copper,
+                                                                          total_zinc_load=curr_total_zinc,
+                                                                          surface_type=surface_type)
+
+        all_roads.iloc[i] = [all_roads.iloc[i]["Index"], surface_area, surface_type, curr_tss,
+                             curr_total_copper, curr_total_zinc, curr_dissolved_copper, curr_dissolved_zinc]
+
+    # all_result = xr.merge(all_roads, all_buildings)
+    return gpd.GeoDataFrame(pd.concat([all_roads, all_buildings], ignore_index=True))
+
+
+def store_pollution_model_in_database(engine: Engine, results: gpd.GeoDataFrame, scenario_id: int):
+    """
+    Appends the details of which buildings are flooded for a given flood_model_id to the database
+
+    Parameters
+    ----------
+    engine: Engine
+        The sqlalchemy database connection engine
+    results : gpd.GeoDataFrame
+        GeoDataFrame containing a mapping of geospatial features (roads, roofs) to their MEDUSA2.0 outputs for the
+        current model run
+    scenario_id : int
+        The id of the current medusa2.0 model run, to associate with the results.
+
+    Returns
+    -------
+    None
+        This function does not return anything
+    """
+    results["scenario_id"] = scenario_id
+    results.set_index("Index", inplace=True)
+    results.to_sql("medusa2_model_output", engine, if_exists="append", index=True)
+    # Create geoserver endpoints for database views if they do not already exist I noticed this was in the code for a
+    # similar function, so I've added it. Please comment if this is necessary or not @Luke
+    create_building_database_views_if_not_exists()
```

Comment:
```suggestion
```
This isn't quite the right function to do it, but it isn't needed for this PR so we can resolve it in a PR for issue #160